app/src/main/java/org/schabi/newpipe/player/helper/PlayerSemitoneHelper.java

```diff
@@ -0,0 +1,37 @@
+package org.schabi.newpipe.player.helper;
+
+/**
+ * Converts between percent and 12-tone equal temperament semitones.
+ * <br/>
+ * @see
+ * <a href="https://en.wikipedia.org/wiki/Equal_temperament#Twelve-tone_equal_temperament">
+ *     Wikipedia: Equal temperament#Twelve-tone equal temperament
+ * </a>
+ */
+public final class PlayerSemitoneHelper {
+    public static final int TONES = 12;
```

Comment:
I'd rename this to `SEMITONE_COUNT` or something like that, since `TONES` is different from `SEMITONES` (a tone is twice a semitone)